### PR TITLE
fix mantl url typo

### LINKED ISSUE #192: link to Project Mantl from Cisco contains extra 'http://'
The link to "Project Mantl from Cisco" contains a small error:

```
<li>Project <a href="http://http://mantl.io/">Mantl</a> from Cisco</li>
```

### PR DESCRIPTION
**Pull request by dontrebootme**
fixes #192

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Refer to the [benchmarks section](docs/index.md#benchmarks) in the documentation
 
 These projects use Træfɪk internally. If your company uses Træfɪk, we would be glad to get your feedback :) Contact us on [![Join the chat at https://traefik.herokuapp.com](https://img.shields.io/badge/style-register-green.svg?style=social&label=Slack)](https://traefik.herokuapp.com)
 
-- Project [Mantl](http://http://mantl.io/) from Cisco
+- Project [Mantl](http://mantl.io/) from Cisco
```
Comment (vdemeester) [inline README.md]: Should we use `https` there ? :angel:

**Comment by emilevauge** [comment]
LGTM, but needs a rebase ;)

**Comment by dontrebootme** [comment]
Updated to https to please @vdemeester and rebased

**Comment by vdemeester** [comment]
thanks @dontrebootme !
LGTM

**Pull request merged**